frontend/src/lib/UriUtil.ts

```diff
@@ -165,13 +165,41 @@ export function buildMediaUri(
 }
 
 /**
- * Check if the given url follows the url pattern which could include
- * a wildcard.
+ * Check if the given origin follows the allowed origin pattern, which could
+ * include wildcards.
```

Comment:
Could you extend this docstring a bit to include details about when the function should be called and what should happen if it returns false? I.e. "you must always call `isValidOrigin` before doing X. If `isValidOrigin` returns false, you must do Y"

Reply:
👍 done